library/core/tests/slice.rs

```diff
@@ -2504,3 +2504,17 @@ fn test_slice_from_ptr_range() {
         assert_eq!(slice::from_ptr_range(range), &arr);
     }
 }
+
+#[test]
+#[should_panic = "slice len overflow"]
+fn test_flatten_size_overflow() {
+    let x = &[[(); usize::MAX]; 2][..];
+    let _ = x.flatten();
+}
+
+#[test]
```

Comment:
```suggestion
#[cfg(not(bootstrap))]
#[test]
```

Reply:
It looks like you might not have pulled before squashing and force-pushing, since these disappeared again.